frontend/src/components/RepoSourceDot.css

```
@@ -0,0 +1,62 @@
+  outline: 1px solid var(--border);
+  outline-offset: 2px;
+}
+
+.repo-source-dot__lock {
+  position: absolute;
+  right: -5px;
+  bottom: -3px;
+  width: 7px;
+  height: 7px;
+  color: var(--text-muted);
+  stroke: currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)